src/main/java/roomescape/service/ReservationTimeService.java

```diff
@@ -0,0 +1,46 @@
+package roomescape.service;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.domain.ReservationTime;
+import roomescape.exception.ReservationTimeNotFoundException;
+import roomescape.infrastructure.JdbcReservationTimeRepository;
+import roomescape.service.dto.ReservationTimeDto;
+
+@Transactional
+@Service
+public class ReservationTimeService {
+
+  private final JdbcReservationTimeRepository reservationTimeRepository;
```

Comment:
```suggestion
  private final ReservationTimeRepository reservationTimeRepository;
```
인터페이스를 필드 타입으로 사용하는 것이 변경에 용이할 것 같아요

Reply:
변경에 용이하도록 인터페이스로 구현했는데 쓸모 없게 만들었네요ㅋㅋㅋ